test/library/packages/LinearAlgebra/correctness/no-dependencies/solveLU.chpl

```diff
@@ -35,7 +35,8 @@ var res = true;
 
     // solve
     const x = solve(Ao, bo);
-    bo = b; // `solve` overrites bo
+    assertAlmostEqual(b, bo, "solve() modified its argument");
+    bo = b;
```

Comment:
Since we've fixed it, do we still need to explicitly assign into `bo`?

Reply:
No, we don't. What was I thinking? I removed it and rebased.

Reply:
Heh, it's easy to do!